.gha-reversemap.yml

```diff
@@ -0,0 +1,85 @@
+actions/checkout:
+  sha: 11bd71901bbe5b1630ceea73d27597364c9af683
+  sha-url: https://github.com/actions/checkout/commit/11bd71901bbe5b1630ceea73d27597364c9af683
+  tag: v4
+  tag-url: https://github.com/actions/checkout/tree/v4
+actions/add-to-project:
+  sha: 244f685bbc3b7adfa8466e08b698b5577571133e
+  sha-url: https://github.com/actions/add-to-project/commit/244f685bbc3b7adfa8466e08b698b5577571133e
+  tag: v1.0.2
+  tag-url: https://github.com/actions/add-to-project/tree/v1.0.2
+titoportas/update-project-fields:
+  sha: 421a54430b3cdc9eefd8f14f9ce0142ab7678751
+  sha-url: https://github.com/titoportas/update-project-fields/commit/421a54430b3cdc9eefd8f14f9ce0142ab7678751
+  tag: v0.1.0
+  tag-url: https://github.com/titoportas/update-project-fields/tree/v0.1.0
+ScholliYT/Broken-Links-Crawler-Action:
+  sha: a074e6a0c83f29560300752af773c7356041ceb4
+  sha-url: https://github.com/ScholliYT/Broken-Links-Crawler-Action/commit/a074e6a0c83f29560300752af773c7356041ceb4
+  tag: fix-http-redirects
+  tag-url: https://github.com/ScholliYT/Broken-Links-Crawler-Action/tree/fix-http-redirects
+JasonEtco/create-an-issue:
+  sha: 1b14a70e4d8dc185e5cc76d3bec9eab20257b2c5
+  sha-url: https://github.com/JasonEtco/create-an-issue/commit/1b14a70e4d8dc185e5cc76d3bec9eab20257b2c5
+  tag: v2
+  tag-url: https://github.com/JasonEtco/create-an-issue/tree/v2
+actions/setup-python:
+  sha: a26af69be951a213d495a4c3e4e4022e16d87065
+  sha-url: https://github.com/actions/setup-python/commit/a26af69be951a213d495a4c3e4e4022e16d87065
+  tag: v5
+  tag-url: https://github.com/actions/setup-python/tree/v5
+actions/setup-go:
+  sha: 0aaccfd150d50ccaeb58ebd88d36e91967a5f35b
+  sha-url: https://github.com/actions/setup-go/commit/0aaccfd150d50ccaeb58ebd88d36e91967a5f35b
+  tag: v5
+  tag-url: https://github.com/actions/setup-go/tree/v5
+anchore/sbom-action/download-syft:
+  sha: 9f7302141466aa6482940f15371237e9d9f4c34a
+  sha-url: https://github.com/anchore/sbom-action/commit/9f7302141466aa6482940f15371237e9d9f4c34a
+  tag: v0.19.0
+  tag-url: https://github.com/anchore/sbom-action/tree/v0.19.0
+goreleaser/goreleaser-action:
+  sha: 9c156ee8a17a598857849441385a2041ef570552
+  sha-url: https://github.com/goreleaser/goreleaser-action/commit/9c156ee8a17a598857849441385a2041ef570552
+  tag: v6
+  tag-url: https://github.com/goreleaser/goreleaser-action/tree/v6
+azure/setup-helm:
+  sha: 37dd2562cae2f186a3dc13f2bb5d4abe08dbfec4
+  sha-url: https://github.com/azure/setup-helm/commit/37dd2562cae2f186a3dc13f2bb5d4abe08dbfec4
+  tag: v4
+  tag-url: https://github.com/azure/setup-helm/tree/v4
+docker/login-action:
+  sha: 74a5d142397b4f367a81961eba4e8cd7edddf772
+  sha-url: https://github.com/docker/login-action/commit/74a5d142397b4f367a81961eba4e8cd7edddf772
+  tag: v3
+  tag-url: https://github.com/docker/login-action/tree/v3
+actions/first-interaction:
+  sha: fb2402657b4a28582200150d0a145671d0e50597
```

Comment:
I do not see this hash listed at https://github.com/actions/first-interaction/tags